src/bootstrap/src/core/download.rs

```diff
@@ -258,18 +258,14 @@ impl Config {
             // attempts will be made, since the first attempt isn't a *re*try.
             "--retry",
             "3",
-<<<<<<< HEAD
-            "-Sf",
-            "-L", // Follow redirects
-            "-SRf",
-=======
```

Comment:
It's fine to remove those, they are already covered by the other provided flags.